Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -3,213 +3,5 @@ import ApolloCodegenTestSupport
-                                       urlToSchemaFile: starWarsSchemaFileURL,
-                                       downloadTimeout: CodegenTestHelper.timeout)
-    do {
-      _ = try ApolloCodegen.run(from: starWarsFolderURL,
-                                with: scriptFolderURL,
-                                options: options)
-    } catch {
-      switch error {
-      case ApolloCodegen.CodegenError.singleFileButNotSwiftFileURL(let url):
-        XCTAssertEqual(url, outputFolder)
-      default:
-        XCTFail("Unexpected error running codegen: \(error.localizedDescription)")
-      }
-    }
-  }
-  
-  func testSingleFileOutputFormat_givenAFileURL_shouldSucceed() throws {
-    let scriptFolderURL = CodegenTestHelper.cliFolderURL()
-    let starWarsFolderURL = CodegenTestHelper.starWarsFolderURL()
-    let starWarsSchemaFileURL = CodegenTestHelper.starWarsSchemaFileURL()
-    let outputFolder = CodegenTestHelper.outputFolderURL()
-    let outputFile = outputFolder.appendingPathComponent("API.swift")
-    
-    let options = ApolloCodegenOptions(outputFormat: .singleFile(atFileURL: outputFile),
-                                       urlToSchemaFile: starWarsSchemaFileURL,
-                                       downloadTimeout: CodegenTestHelper.timeout)
-    do {
-      _ = try ApolloCodegen.run(from: starWarsFolderURL,
-                                with: scriptFolderURL,
-                                options: options)
-    } catch {
-      XCTFail("Error running codegen: \(error.localizedDescription)")
-      return
-    }
-    
-    XCTAssertTrue(FileManager.default.apollo.folderExists(at: outputFolder))
-    XCTAssertTrue(FileManager.default.apollo.fileExists(at: outputFile))
-    
-    let contents = try FileManager.default.contentsOfDirectory(atPath: outputFolder.path)    
-    XCTAssertEqual(contents.count, 1)
-  }
-  
-  func testMultipleFilesOutputFormat_givenAFolderURL_shouldSucceed() throws {
-    let scriptFolderURL = CodegenTestHelper.cliFolderURL()
-    let starWarsFolderURL = CodegenTestHelper.starWarsFolderURL()
-    let starWarsSchemaFileURL = CodegenTestHelper.starWarsSchemaFileURL()
-    let outputFolder = CodegenTestHelper.outputFolderURL()
-    
-    let options = ApolloCodegenOptions(outputFormat: .multipleFiles(inFolderAtURL: outputFolder),
-                                       urlToSchemaFile: starWarsSchemaFileURL,
-                                       downloadTimeout: CodegenTestHelper.timeout)
-    
-    _ = try ApolloCodegen.run(from: starWarsFolderURL,
-                              with: scriptFolderURL,
-                              options: options)
-    
-    XCTAssertTrue(FileManager.default.apollo.folderExists(at: outputFolder))
-    
-    let contents = try FileManager.default.contentsOfDirectory(atPath: outputFolder.path)
-    XCTAssertEqual(contents.count, 18)
-  }
+  #warning("TODO: Write extensive tests for codegen folder/module structure.")
```

Comment:
is it worth leaving the test names with `XCTSkip` to make sure we have equivalent coverage?

Reply:
I've instead opted to leave the test names as suggested but make them all fail with `XCTFail`. Using `XCTSkip` on the tests that don't throw is effectively the same as just leaving them empty and not serving as a hard-stop. Thanks for the great suggestion!